aqua-ethos-setup/aqua_ethos.sh

```diff
@@ -85,128 +70,88 @@ function login {
-EOF
-
-EXISTING_PROFILE=$(makeGet securityprofiles/Ethos)
-
-# TODO: check for same env var enc status
-
-PROFILE_BODY="{\"name\": \"Ethos\", \"enforce\": true, \"type\": \"security.profile\", \"description\": \"Ethos Default RunTime Profile\", \"encrypt_all_envs\": $ENC_ENV_VARS, \"seccomp_profile\": \"$SECCOMP_PROFILE_JSON\"}"
-
-if [[ "$EXISTING_PROFILE" == "200" ]]; then
-	log "Ethos profile exists..."
-
-	PROFILE_CONTENTS=$(makeGet securityprofiles/Ethos body)
-
-	ENC_ENABLED=$(echo "$PROFILE_CONTENTS" | jq '.encrypt_all_envs')
-
-	if [[ "$ENC_ENABLED" != "$ENC_ENV_VARS" ]]; then
-		log "WARNING: Ethos profile does not match ENC_ENV_VARS setting."
-		# makePut "securityprofiles" "$PROFILE_BODY"
-	else
-		log "Ethos profile matches ENC_ENV_VARS setting..."
-	fi
-else
-	makePost "securityprofiles" "$PROFILE_BODY"
-fi
-
-# See if artifactory integration already exists
-if [[ ! -z "$ARTIFACTORY_URL" ]]; then
-	log "ARTIFACTORY_URL set. Checking artifactory configs..."
-
-	EXISTING_ARTIFACTORY=$(makeGet "registries/artifactory-test")
-
-	if [[ "$EXISTING_ARTIFACTORY" == "200" ]]; then
-		log "Artifactory integration exists..."
-	else
-		makePost "registries" '{"prefixes": [], "auto_pull": false, "auto_pull_time": "03:00", "auto_pull_max": 100, "name": "artifactory-test", "type": "ART", "webhook": {"enabled": false, "url": "", "auth_token": ""}, "url": "'$ARTIFACTORY_URL'", "username": "'$ARTIFACTORY_USERNAME'", "password": "'$ARTIFACTORY_PASSWORD'"}'
-	fi
-fi
+function replaceConfigs {
+	log "Replacing ETH configs in $CONFIG_FILE"
+
+	# Note: using "@" instead of "/" as delimiter because some expressions contain slashes (URLs)
+	sed -i.bak "s@ETH_ARTIFACTORY_URL@$ARTIFACTORY_URL@g" "$CONFIG_FILE"
+	sed -i.bak "s@ETH_ARTIFACTORY_PREFIX@$ARTIFACTORY_PREFIX@g" "$CONFIG_FILE"
```

Comment:
do these need to be inside of `{}` like  `${ARTIFACTORY_PREFIX}` just in case there are complex characters?